### PR TITLE
Added wkhtmltopdf_args to render_template_to_pdf

### PR DESCRIPTION
**Pull request by garrosh**
So we can now give a list of options to pass for the call to wkhtmltopdf

### CODE REVIEW

File: flask_wkhtmltopdf.py [hunk 1]
```diff
@@ -117,8 +118,19 @@ def render_template_to_pdf(self, template_name_or_list, save=False, download=Fal
         with tempfile.NamedTemporaryFile(suffix='.pdf', dir=self.pdf_dir_path, delete=False) as temp_pdf:
             pass
 
+        cli_options = ""
+        
+        # Parse argument list supplied and add them as options to wkhtmltopdf
+        if wkhtmltopdf_args:
+            for argument in wkhtmltopdf_args:
+                print(argument)
```
Comment (chris-griffin) [inline flask_wkhtmltopdf.py]: Can you remove this line?

File: flask_wkhtmltopdf.py [hunk 2]
```diff
@@ -117,8 +118,19 @@ def render_template_to_pdf(self, template_name_or_list, save=False, download=Fal
         with tempfile.NamedTemporaryFile(suffix='.pdf', dir=self.pdf_dir_path, delete=False) as temp_pdf:
             pass
 
+        cli_options = ""
+        
+        # Parse argument list supplied and add them as options to wkhtmltopdf
+        if wkhtmltopdf_args:
+            for argument in wkhtmltopdf_args:
+                print(argument)
+                if argument.startswith('--'):
```
Comment (chris-griffin) [inline flask_wkhtmltopdf.py]: 👍

**Review by chris-griffin — CHANGES_REQUESTED** [review]
If you are able to add a test as well that would be great

**Comment by garrosh** [comment]
I would actually like to, but I haven't done that before, so I'm afraid I'd fudge it up.

**Comment by chris-griffin** [comment]
Thanks for the contribution! I'll do a follow up PR to add the test and fix some of the old ones. I haven't touched this repo in a long time

**Pull request merged**